opencda/core/ml_libs/rl/simulators/carla_simulator.py

```diff
@@ -0,0 +1,790 @@
+'''
```

Comment:
This class has a lot of things overlapped with our original simulator manager in sim_api.py. I don't like this that we have a lot of redundant functions. Rewrite this please